.github/ISSUE_TEMPLATE/🐜bug-report--vs-code-runtime-extension.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F41Cbug report: vs-code runtime extension"
+about: 'bug report: vs-code runtime extension'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Privacy Alert! The contents copied to your clipboard may contain personal data. Prior to posting to GitHub, please remove any personal data which should not be publicly viewable. https://privacy.microsoft.com/en-US/privacystatement`;
+-->
+
+**Describe the bug**
+The issue text was copied to the clipboard.  Please paste it into this window.
+
+Please attach the log file located at ${ context.logger.getFileLocation() }. Note that this file may contain personal data.
```

Comment:
How does the context log file location get filled out in this situation?

Reply:
I haven't tried yet but it might be from the `IssueReport.ts`

Reply:
Tried a couple of hours and appending might not be a solution. Maybe provide a fix path